### PR TITLE
Add support for EmailInvoicingAddress field

### PR DESCRIPTION
**Pull request by tvuotila**
Add support for customer EmailInvoicingAddress for [get](https://netvisor.zendesk.com/hc/fi/articles/201993973-Resources-Customer-register#ankkuri3) and [import](https://netvisor.zendesk.com/hc/fi/articles/201993973-Resources-Customer-register#ankkuri4).

### CODE REVIEW

**Comment by jpvanhal** [comment]
> Specification defines that "type" attribute is written in lower case.

Which specification? XML? Netvisor's API?

File: tests/data/requests/Product.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <ProductGroup>Kirjat</ProductGroup>
     <Name>Code Complete</Name>
     <Description>Toinen painos</Description>
-    <UnitPrice Type="brutto">42,5</UnitPrice>
+    <UnitPrice type="brutto">42,5</UnitPrice>
```
Comment (jpvanhal) [inline tests/data/requests/Product.xml:7]: Is `type` lowercase also in Netvisor's API responses?

**Comment by jpvanhal** [comment]
> xmltodict v0.10.1 added postprocessor support for attributes.
> As result, attribute names are now formated.

If we depend on some functionality added in v0.10.1, we should reflect this requirement to `setup.py`.

**Comment by tvuotila** [comment]
> Specification defines that "type" attribute is written in lower case.
> Which specification? XML? Netvisor's API?

[Netvisor specification](https://netvisor.zendesk.com/hc/fi/articles/202363038-Resources-Products-and-Warehouse#ankkuri2)

> Is type lowercase also in Netvisor's API responses?

Yes, it is.

> If we depend on some functionality added in v0.10.1, we should reflect this requirement to setup.py.

Because "Type" is "type" in Netvisor response. We don't need to depend on specific version. I will update the commit message to reflect this.

**Comment by tvuotila** [comment]
@jpvanhal Do you have time to look this again?

**Comment by jpvanhal** [comment]
Yep, merged and published v0.6.0.

**Pull request merged**